app.js

```diff
@@ -0,0 +1,17 @@
+// Importing modules
+const express = require('express');
+const path = require('path');
+var favicon = require('serve-favicon')
+const app = express();
+
+app.get('/', (req, res) => {
+    res.sendFile(path.join(__dirname, '/public/home.html'));
+});
+
+
+app.use(favicon(path.join(__dirname, '/public', 'favicon.ico')))
+app.use(express.static(path.join(__dirname, '/public')));
+
+app.listen(process.env.PORT || 4000, function () {
+    console.log('Node app is working!');
```

Comment:
It's convenient sometimes to add a port number to this line. It helps with checking env configuration and with opening an appropriate URL in a browser when you launch the project after a while.